core/test/src/org/labkey/test/tests/ProjectWizardTest.java

```diff
@@ -0,0 +1,79 @@
+package org.labkey.test.tests;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.Git;
+import org.labkey.test.components.BodyWebPart;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Git.class})
+public class ProjectWizardTest extends BaseWebDriverTest
+{
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        super.doCleanup(afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        ProjectWizardTest init = (ProjectWizardTest) getCurrentTest();
+
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        log("this test verifies the project creation wizard from the project menu");
+    }
+
+    @Before
+    public void preTest() throws Exception
+    {
+        goToHome();
+    }
+
+    @Test
+    public void testCreateFromProjectMenu() throws Exception
+    {
+        projectMenu()
+                .navigateToCreateProjectPage()
+                .setFolderType("Collaboration")
+                .setProjectName(getProjectName())
+                .clickNext()
+                .setMyUserOnly()
+                .clickNext()
+                .useDefaultLocation()
+                .clickFinish();
+
+        // confirm expected webparts in a collaboration project are present
+        new BodyWebPart(getDriver(), "Subfolders");
```

Comment:
Don't assume that the `BodyWebPart` constructor will actually find the webpart; it might get refactored to not do so at some point. Just grab the actual titles and verify that they contain what you want.
```suggestion
        assertThat("WebParts for collaboration folder type.", new PortalHelper().getWebPartTitles(), hasItem("Subfolders"))
```
_Sidenote: the thought of this is the sort of pattern is what makes me worry about helpers that return `null` or `-1` when something isn't found._

Reply:
Huh.  Didn't realize getWebPartTitles() existed, thanks for pointing it out